test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -108,9 +108,12 @@ const PossibleIssues = () => {
         if (buildName.includes('j9') || buildName.includes('ibm')) {
             additionalRepo = '+repo:eclipse-openj9/openj9';
         }
+        else if (buildName.includes('hs')){
```

Comment:
Please check if you have `Prettier` installed in your IDE. The code format is not auto-corrected.